### ISSUE TITLE
Need help getting oAuth boilerplate to function

### ISSUE THREAD

**Issue by smirciat** [DaftMonk/generator-angular-fullstack#1102]
I ran the generator with defaults and have not customized anything.  I have good functionality of the base application.  Clicking on Facebook, Google, and Twitter to signup or login throws an authentication error for each one.  Is there any additional guidance for making these work?  I can get the local authorization to work right out of the box.  I did update code to match #1062 but it did not help.
UPDATE: Looks like #942 answers this, I am trying it out.

**Comment by kingcody** [comment]
@smirciat any luck? Were you able to resolve your issues by adding the `callbackURL` to your oauth providers, as well as filling out your `clientID` and `clientSecret` (in `server/config/local.env.js`) for the specific providers?

**Comment by smirciat** [comment]
@kingcody thanks so much for the follow up.  I am still not connecting to the oAuth services.  For example, with Twitter, I set up an app at apps.twitter.com with the base url of my application and the callback with the same url plus the extension /auth/twitter/callback.  This is on Heroku deployment. I entered the API key and API secret as supplied by this service in server/config/local.env.js as specified.  I have no other customizations yet from the yo angular-fullstack generation.  What might I be missing?
